src/lib/components/globalhealthbar/GlobalHealthBar.component.js

```diff
@@ -0,0 +1,139 @@
+//@flow
+import React from 'react';
+import VegaChart from '../vegachart/VegaChart.component.js';
+
+type Props = {
+  id: string,
+  alertSegments: {
+    severity: 'critical' | 'warning' | 'none',
+    startSegment: string,
+    endSegment: string,
+    description: string,
+  }[],
+  start: { date: number, month: number, year: number },
+  end: { date: number, month: number, year: number },
+  height?: number,
+  width?: number,
+  theme?: 'light' | 'dark' | 'custom',
+  tooltipPosition?: 'up' | 'down',
+};
+
+function GlobalHealthBar({
+  id,
+  alertSegments,
+  start,
+  end,
+  height = 16,
+  width = 650,
+  theme = 'dark',
+  tooltipPosition = 'up',
+}: Props) {
+  const spec = {
+    width,
+    height,
+    data: {
+      values: alertSegments,
+    },
+    transform: [
+      {
+        calculate:
+          "datum.description !== '' ? 'View details on alart page' : ''",
+        as: 'title',
+      },
+    ],
+    layer: [
+      // Paint the entire bar with green
+      {
+        mark: { type: 'rect', cornerRadius: 15 },
+        encoding: { color: { value: '#28a745' } },
```

Comment:
We should use a well known theme color (https://github.com/scality/core-ui/issues/259)